README.md

```diff
@@ -66,4 +66,4 @@ There are many .NET projects on GitHub.
 [Introducing .NET Core]: http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx
 [coreclr blog post]: http://blogs.msdn.com/b/dotnet/archive/2015/02/03/coreclr-is-open-source.aspx
 [corefx]: http://github.com/dotnet/corefx
-[coreclr]: http://github.com/dotnet/coreclr
\ No newline at end of file
+[coreclr]: http://github.com/dotnet/coreclr
```

Comment:
I used Github's edit button to edit the file. It appears that a post-commit hook in github removed a trailing newline.